Detectors/TOF/prototyping/ConvertRun2CalibrationToO2.C

```diff
@@ -76,12 +77,17 @@ void ConvertRun2CalibrationToO2()
 
   Float_t x[10000];
```

Comment:
what defines these 10000? Here you allocate 160KB on the stack. Normally should not be critical, but is it needed?

Reply:
This is a macro used to create O2 CCDB-like objects from Run2-OCDB objects. It does not need to be optimized. 10000 is simply to be sure to have enough. I would leave it as it is.

Reply:
ok, sure